omod/src/main/webapp/pages/login.gsp

```diff
@@ -1,6 +1,7 @@
 <%
     ui.includeFragment("appui", "standardEmrIncludes")
     ui.includeCss("referenceapplication", "login.css")
+    ui.includeCss("uicommons", "form-navigator-ui.css");
```

Comment:
@dkayiwa i tried to follow the pattern i saw in other files and tried including form-navigator-ui.css even thought there is only a form-navigator-ui.scss, but it still did not work, as a result, the style specified for .field-error is not applied. :(